### ISSUE TITLE
plotProfile() ignores 'mar[1]' value

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1103]
For example
```R
library(oce)
data(ctd)
par(mfrow=c(1,2))
plotProfile(ctd,"density")
plotProfile(ctd,"density",mar=c(5,3.5,4,1))
```

**Comment by dankelley** [comment]
I see why (screenshot of `R/ctd.R`) but will need to think of just why I did this, before changing it.
![screen shot 2016-10-20 at 10 01 40 am](https://cloud.githubusercontent.com/assets/99469/19560821/547232c2-96ac-11e6-98b6-1587d0add759.png)

**Comment by dankelley** [comment]
I see why I did that -- it prevents the large bottom space that would be used if the default were to assume axes at both top and bottom. The underlying problem is that the code is using a default that is stated in the function argument list, and that prevents inference of whether the user supplied the argument or not. I will change this so that the argument is by default not specified. Then, (a) I can check if it's `missing` in the code block I've illustrated in the previous comment and (b) the argument list will be cleaner.  (This second item is something I want to do generally. It takes a bit more work in writing the documentation, but I think it will be clearer to the user, since it will shorten the length of that argument list.)
